docs/install_apt_debian.md

```diff
@@ -4,10 +4,11 @@ title: "Installation: Debian"
 
 # Debian Installation
 
-Caffe packages are available for `Debian/unstable`. Debian/stable users
-should take a look at Ubuntu installation instruction.  
+Caffe packages are available for `Debian/unstable` (Debian 10). Debian/stable
+(jessie) and Debian/testing (stretch) users should take a look at [Ubuntu
```

Comment:
@CDLuminate I thought that I would be able to edit or revert my commit to your PR, but it looks like I can only make another commit changing my previous one. I think it would be good to at least mention that by "testing", you mean "stretch", but the other changes are not necessary. Would you mind reverting my commit? Sorry for the hassle.